wiki/Beatmap/Genre_and_language/de.md

```diff
@@ -0,0 +1,67 @@
+
+- Beatmap-Eigentümer dürfen das, wenn sich die Beatmap in der Kategorie [WIP](/wiki/Beatmap/Category#work-in-progress-und-ausstehend), [Ausstehend](/wiki/Beatmap/Category#work-in-progress-und-ausstehend) oder [Friedhof](/wiki/Beatmap/Category#friedhof) befindet und keine [Nominierungen](/wiki/Beatmap_ranking_procedure#nominations) hat.
+- [Beatmap Nominator](/wiki/People/Beatmap_Nominators) dürfen das, wenn sich die Beatmap in der Kategorie [Qualifiziert](/wiki/Beatmap/Category#qualifiziert), WIP oder Ausstehend befindet.
+- Mitglieder des [Project Loved Teams](/wiki/People/Project_Loved_Team) dürfen das, wenn sich die Beatmap in der Kategorie [Loved](/wiki/Beatmap/Category#loved) befindet.
+- Mitglieder des [Nomination Assessment Teams](/wiki/People/Nomination_Assessment_Team) und des [Globalen Moderationsteams](/wiki/People/Global_Moderation_Team) dürfen das für alle Kategorien.
+
+Eine Beatmap kann nicht [nominiert](/wiki/Beatmap_ranking_procedure#nominations) werden, wenn die Sprache oder das Genre auf `Nicht angegeben` eingestellt ist.
+
+## Liste der Genres
+
+| Genre | Merkmale entsprechender Songs |
+| :-- | :-- |
+| Nicht angegeben | Keine Songs treffen zu. Dieses Genre ist ein Platzhalter, bis ein geeignetes Genre festgelegt ist. |
+| Videospiel | Gemacht für oder populär geworden durch ein Videospiel. Dies wird auch für Arrangements und Remixes dieser Songs verwendet. |
+| Anime | Produziert für oder bekannt geworden durch einen Anime oder ein vergleichbares Medium. |
+| Rock | Mit einem Schwerpunkt auf Gitarren, Schlagzeug und Bass, typischerweise auf der Grundlage von einfachen synkopischen Rhythmen basierend und von Live-Auftritten und einem Fokus auf ernste sowie progressive Motive charakterisiert ([*Rockmusik* auf Wikipedia](https://de.wikipedia.org/wiki/Rockmusik)). |
+| Pop | In der Popkultur weit verbreitet und durch die Verwendung einer einfachen Struktur und wiederholter Refrains oder Melodien eingängig komponiert. |
+| Andere | Keinem der aufgeführten Genres zuzuordnen. |
+| Novelty | Lustig oder ungewöhnlich. In osu! beinhaltet diese Kategorie hauptsächlich Songs wie "YouTube Poops", "Niconico MADs" und Meme-Musik. |
+| Hip-Hop | Stilisierte rhythmische Musik, die häufig das Rappen begleitet und manchmal Samples oder Bassaufnahmen verwendet ([*Hip-Hop* auf Wikipedia](https://de.wikipedia.org/wiki/Hip-Hop)). |
+| Electronic | Digital oder mit elektronischen Instrumenten komponiert. |
+| Metal | Ausdrucksstarker Sound mit lauten, verzerrten E-Gitarren und Bässen sowie schnellen oder rasanten Schlagzeugrhythmen. |
+| Klassik | Formal, auf der Grundlage fortgeschrittener Musiktheorie strukturiert und in der alten westlichen Kultur verwurzelt oder von ihr inspiriert. |
+| Folk | Traditionell oder persönlich, leger und die Identität einer Kultur oder einer Gruppe von Menschen widerspiegelnd. |
```

Comment:
If `leger` is something you find too uncommon, `locker` might be an alternative.

Reply:
I think `locker` works probably better for this. I don't think I ever heard `leger` in this context.

Reply:
I prefer `locker` too. Maybe add a comma after `leger`/`locker` (whichever you'll use).

Reply:
```suggestion
| Folk | Traditionell oder persönlich, locker und die Identität einer Kultur oder einer Gruppe von Menschen widerspiegelnd. |
```

I don't think a comma should be put there.